models/ellipsis_finance/bnb/ellipsis_finance_bnb_sources.yml

```diff
@@ -0,0 +1,202 @@
+    tables:
+      - name: 2brl_call_coins
+        loaded_at_field: call_block_time
+      - name: 2brl_evt_TokenExchange
+        loaded_at_field: evt_block_time
+      - name: 2pool_call_coins
+        loaded_at_field: call_block_time
+      - name: 2pool_evt_TokenExchange
+        loaded_at_field: evt_block_time
+      - name: 3brl_call_coins
+        loaded_at_field: call_block_time
+      - name: 3brl_evt_TokenExchange
+        loaded_at_field: evt_block_time
+      - name: 3EPS_call_coins
+        loaded_at_field: call_block_time
+      - name: 3EPS_evt_TokenExchange
+        loaded_at_field: evt_block_time
+      - name: Ankr_BNB_call_coins
+        loaded_at_field: call_block_time
+      - name: Ankr_BNB_evt_TokenExchange
+        loaded_at_field: evt_block_time
+      - name: ankr_eth_call_coins
+        loaded_at_field: call_block_time
+      - name: ankr_eth_evt_TokenExchange
+        loaded_at_field: evt_block_time
+      - name: ankr_matic_call_coins
+        loaded_at_field: call_block_time
+      - name: ankr_matic_evt_TokenExchange
+        loaded_at_field: evt_block_time
+      - name: apl_BUSD_call_coins
+        loaded_at_field: call_block_time
+      - name: apl_BUSD_evt_TokenExchange
+        loaded_at_field: evt_block_time
+      - name: ARTH_usd_call_coins
+        loaded_at_field: call_block_time
+      - name: ARTH_usd_evt_TokenExchange
+        loaded_at_field: evt_block_time
+      - name: ARTH_usd_evt_TokenExchangeUnderlying
+        loaded_at_field: evt_block_time
+      - name: AUSD_3EPS_call_coins
+        loaded_at_field: call_block_time
+      - name: AUSD_3EPS_evt_TokenExchange
+        loaded_at_field: evt_block_time
+      - name: AUSD_3EPS_evt_TokenExchangeUnderlying
+        loaded_at_field: evt_block_time
+      - name: axelarUSD_call_coins
+        loaded_at_field: call_block_time
+      - name: axelarUSD_evt_TokenExchange
+        loaded_at_field: evt_block_time
+      - name: bnb_bnbl_call_coins
+        loaded_at_field: call_block_time
+      - name: bnb_bnbl_evt_TokenExchange
+        loaded_at_field: evt_block_time
+      - name: BNBx_BNB_call_coins
+        loaded_at_field: call_block_time
+      - name: BNBx_BNB_evt_TokenExchange
+        loaded_at_field: evt_block_time
+      - name: cryptopool_BNBx_BNB_call_coins
+        loaded_at_field: call_block_time
+      - name: cryptopool_BNBx_BNB_evt_TokenExchange
+        loaded_at_field: evt_block_time
+      - name: cryptopool_BUSD_ARTH_bsc_call_coins
+        loaded_at_field: call_block_time
+      - name: cryptopool_BUSD_ARTH_bsc_evt_TokenExchange
+        loaded_at_field: evt_block_time
+      - name: crypto_pool_BUSD_BTCB_call_coins
+        loaded_at_field: call_block_time
+      - name: crypto_pool_BUSD_BTCB_evt_TokenExchange
+        loaded_at_field: evt_block_time
+      - name: crypto_pool_BUSD_DDD_call_coins
+        loaded_at_field: call_block_time
+      - name: crypto_pool_BUSD_DDD_evt_TokenExchange
```

Comment:
These look like all events from what I would assume are the same contracts (but different instance deployments).
Ideally we get those in 1 event table and we can just use that as source and differentiate between the pools by the contract_address. Have you indicated these were contracts generated by a factory when submitting these events for decoding?